### ISSUE TITLE
Ability to move multiple albums at once

### ISSUE THREAD

**Issue by CChiappa** [jradwan/Piwigo-physical_photo_move#17]
I typically move albums at the end of a year to a subalbum specific to that year.  ie, A-2017, B-2017 gets moved to 2017/ at the end of the year.  It would be helpful to be able to grab a bunch of albums at once and move them all in batch.  However, it looks like piwigo doesn't currently have a facility for bulk operations on albums, so this is probably a long range wishlist item :)

**Comment by jradwan** [comment]
Yeah, not sure right now how I would accomplish that via Batch Manager. Personally I just initially put my albums under the year to start with (i.e., 2018/January, 2018/February) but I can see where a batch move could be useful. Marked as enhancement for future consideration. 😄

**Comment by miblo69** [comment]
I can second this feature request. It would be very helpful when re-organizing the albums.
